setup.py

```diff
@@ -65,11 +65,21 @@
         'numpydoc'],
     'examples': [
         'jupyter',
-        'matplotlib',
-        'plotly']
+        'pandas',
+        'openml']
 }
 
 
+def combine_requirements(new_key, base_keys):
+    EXTRAS_REQUIRE[new_key] = list(
+        set(k for v in base_keys for k in EXTRAS_REQUIRE[v]))
```

Comment:
Minor comment maybe make this function return this list,
and then use it as,
```py
EXTRAS_REQUIRE["dev"] = combine_requirements([k for k in EXTRAS_REQUIRE if k != "examples"])
```
that might be a bit easier to read.

Reply:
good idea @rth - i've made the change you suggested